config/v1/types_image.go

```diff
@@ -25,25 +25,29 @@ type ImageSpec struct {
 	// permission to create Images or ImageStreamMappings via the API are not affected by
 	// this policy - typically only administrators or system integrations will have those
 	// permissions.
+	// +optional
```

Comment:
@bparees @enj these changes here in `types_image.go` do not change serialization behaviour. But please double check you really meant what you specified with tags (and fix in a follow-up if you haven't).